lhotse/features/base.py

```diff
@@ -565,14 +564,6 @@ def store_feature_array(
         might be appropriate for non-log space features.
     :return: a path to the file containing the stored array.
```

Comment:
Now it seems to store a storage_key?  Is this concept defined anywhere?

Reply:
Yeah, I will describe it later in the code, the short story is: a single HDF file has a `storage_path` on the filesystem (e.g. `feats.hdf`) and stores multiple utterances indexed by the `storage_key` (e.g. `utt1`, `utt2`, `utt3`). For arrays stored in separate files on the filesystem, `storage_path` would be the name of the directory, and `storage_key` would be the filename inside that directory.